src/util.js

```diff
@@ -29,17 +29,12 @@ async function getConfig () {
       username: 'root',
       password: 'qwerty'
     },
-    slaveServer: {
-      host: '192.168.100.1',
+    xenServer3: {
```

Comment:
Why do you have so many servers?

Their names should express their differences.